remotespark/livyclientlib/pandasscalalivyclient.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+import pandas as pd
+import json
+import re
+
+from .log import Log
+from .livyclient import LivyClient
+
+
+class PandasScalaLivyClient(LivyClient):
```

Comment:
I thought about it too and refrained from doing it just yet. I wanted to hold off until we have a good idea of how Altair integration will look like.